Gemfile.jruby-3.1.lock.release

```diff
@@ -2,7 +2,7 @@ PATH
   remote: logstash-core-plugin-api
   specs:
     logstash-core-plugin-api (2.1.16-java)
-      logstash-core (= 8.17.3)
+      logstash-core (= 8.17.4)
```

Comment:
We have also (8.17.3-java) version definition on L10 in this file.

Reply:
🤦